src/server/components/dashboard/controllers.ts

```diff
@@ -138,85 +135,6 @@ export async function usersEditPostController(req: Request, res: Response, next:
   return res.redirect("/dashboard/users");
 }
 
-export async function listsController(req: Request, res: Response, next: NextFunction): Promise<void> {
```

Comment:
between all of the new functions, as well as a validate function I've added, controllers.ts was getting pretty long. Time for it's own file

Reply:
I've done it in a few places, but it's something you'd find in OOP. 

My other solution would be to 
1. create a directory `/controllers`
2. create a file `/index.ts` (which is either just what is inside controllers.ts OR split out controllers.ts even further and export them via index.ts)
3. in which case, pretty much none of the other files that import controllers have to change. in js, if you import /some/dir - it will automatically check if there is a file called `dir.ts|js|etc`, and if not, look for /some/dir/index.ts|js|etc

the problem with this is I'd be moving quite a lot of things around for the sake of one file.. I didn't want to bother with that for this ticket tbh.. 

Consider: 
```.ts
import * as controllers from './controllers' // which could be an index, which export different files, or one file that has lots of controllers

// to use each of the controllers, you would do something like
controllers.listController();
controllers.loginController();
```

In an ideal would, I would probably just like loginController to export .get,.post,.put but just required too much work, so yeah I agree it's inconsistent/a bit Frankenstein. 


In other languages with strong OOP, kotlin, java (I think - maybe the intellj IDE does some extra magic as well - so it might be one big class but your IDE renders methods as their own file), swift, it's common to see methods split out by file. 
e.g. `MyClass.swift`, `MyClass+onTapButton.swift`. 

and generally 1 method/function or class per file makes you think in "SOLID", and the added git benefits. e.g. if I want to move something but preserve git history `git mv file` - but if I want to abstract something into it's own file, the history will be lost. and hopefully helps with conflicts.